### PR TITLE
2.2.5-beta1 release

### PR DESCRIPTION
**Pull request by outofambit**
- [x] finalize release notes
- [x] release!

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "2.2.5-beta1": [
+      "[Added] Users can configure Desktop to allows bring or leave changes when switching branches - #7746",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Users can configure Desktop to always bring or leave changes when switching branches - #7746",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+      "[Added] Prompt users to re-authorize if any of their GitHub organizations require them to - #8910",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Prompt users to re-authorize if required by any of their GitHub organizations - #8910",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+      "[Fixed] Recognize pull requests made from forks - #8549",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Recognize if the current branch of a fork has a pull request to the upstream repo - #8549",
```

**Comment by outofambit** [comment]
@billygriffin thank you! edits made!

**Pull request merged**